app/test/app.test.js

```diff
@@ -54,3 +53,17 @@ test('check are the error and status code rigth when the slug is not unique', as
     expect(payload1.errors).toEqual(['The slug already exists']);
     await postModel.deletePost(payload);
 })
+
+test('when post id is null', async () => {
+    let result = request('GET', 'http://localhost:8080/posts/666');
+    expect(result.statusCode).toBe(404);
+    let payload = JSON.parse(result.body);
+    expect(await payload.message).toEqual('post does not exist!');
+})
+
+test('when id exists', async () => {
+    let result = request('GET', 'http://localhost:8080/posts/496');
```

Comment:
Jeśli twój ziomek z teamu odpali te testy na swoim kompie to nie przejdą. Tak samo jak ty je odpalisz za pół roku, też raczej nie przejdą.

Zrób to tak:
1) Dodaj post za pomocą metody `insertPost`, w tym momencie będziesz znała `id` które będziesz mogła użyć w zapytaniu
2) Sprawdź czy API HTTP zwróciło to co miało zwrócić
3) Usuń ten post za pomocą metody `deletePost`

Reply:
poprawione, wczoraj nie wpadłam na to, że mogę użyć adresu zapytania w takiej formie
`http://localhost:8080/posts/${newPost.id}` :-)